src/Compilers/CSharp/Test/Semantic/Semantics/StructConstructorTests.cs

```diff
@@ -1592,24 +1619,72 @@ record struct S3()
 }
 ";
 
-            var expectedDiagnostics = new[]
-            {
+            var comp = CreateCompilation(new[] { source, IsExternalInitTypeDefinition }, parseOptions: TestOptions.Regular10);
+            comp.VerifyDiagnostics(
                 // (2,15): error CS0171: Field 'S1.Y' must be fully assigned before control is returned to the caller
                 // record struct S1()
                 Diagnostic(ErrorCode.ERR_UnassignedThis, "S1").WithArguments("S1.Y").WithLocation(2, 15),
+                // (2,15): error CS8652: The feature 'implicit initialization in struct constructors' is currently in Preview and *unsupported*. To use Preview features, use the 'preview' language version.
+                // record struct S1()
+                Diagnostic(ErrorCode.ERR_FeatureInPreview, "S1").WithArguments("implicit initialization in struct constructors").WithLocation(2, 15),
                 // (7,15): error CS0843: Auto-implemented property 'S2.Y' must be fully assigned before control is returned to the caller.
                 // record struct S2()
                 Diagnostic(ErrorCode.ERR_UnassignedThisAutoProperty, "S2").WithArguments("S2.Y").WithLocation(7, 15),
+                // (7,15): error CS8652: The feature 'implicit initialization in struct constructors' is currently in Preview and *unsupported*. To use Preview features, use the 'preview' language version.
+                // record struct S2()
+                Diagnostic(ErrorCode.ERR_FeatureInPreview, "S2").WithArguments("implicit initialization in struct constructors").WithLocation(7, 15),
                 // (12,15): error CS0843: Auto-implemented property 'S3.X' must be fully assigned before control is returned to the caller.
                 // record struct S3()
-                Diagnostic(ErrorCode.ERR_UnassignedThisAutoProperty, "S3").WithArguments("S3.X").WithLocation(12, 15)
-            };
-
```

Comment:
>internal [](http://example.com/codeflow?start=4&length=8)

Consider reverting the accessibility changes.

Reply:
The members will only print if the accessibility is 'public' here, so it seems like it makes the tests more meaningful to have them this way.